migrator/src/main/java/com/google/phonenumbers/migrator/CommandLineMain.java

```diff
@@ -0,0 +1,102 @@
+package com.google.phonenumbers.migrator;
+
+import java.io.IOException;
+import java.nio.file.Paths;
+import java.util.Scanner;
+import picocli.CommandLine;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Help.Ansi;
+import picocli.CommandLine.Option;
+
+@Command(name = "Migrator Tool", description = "Command line phone number migration tool.\n"
+    + "Please enter a path to a text file containing E.164 phone numbers from the same region or a "
+    + "single E.164 number as well as the corresponding region code to begin migrations.\n")
+public class CommandLineMain {
+
+  @Option(names = {"-n", "--number"}, description = "Single E.164 phone number to migrate")
```

Comment:
Examples for these are really nice to have, especially if format matters. ie. Can we have +12223334444 and +1 (222) 333-4444?